head/src/main/resources/views/fragments/campaign-summary.html

```diff
@@ -32,9 +32,9 @@ <h3 class="float-left pb-3" th:text="${campaign.name}">No Campaign</h3>
         <span class="float-left vertical-divider"></span>
         <div th:unless="${campaign.resolvedZones == null}" th:each="zone: ${campaign.resolvedZones}"
              class="float-left campaign-summary-top-third">
-      <span th:if="${zone.image != null}" class="zone float-left"
-            th:style="'background-image: url(' + ${zone.image}+ '); background-repeat: no-repeat, repeat; background-size: cover;'"></span>
-            <div th:unless="${zone.image != null}" class="zone float-left">
+      <span th:if="${zone.imagePath != null}" class="zone float-left"
+            th:style="'background-image: url(' + ${zone.imagePath}+ '); background-repeat: no-repeat, repeat; background-size: cover;'"></span>
+            <div th:unless="${zone.imagePath != null}" class="zone float-left">
```

Comment:
You checked just above that zone.imagePath is not null. Is that still required here?

Reply:
th:unless="${zone.imagePath is like the `else` block of the statment